### PR TITLE
adding cv topology folder

### PR DESCRIPTION
**Pull request by avkewalr-msft**
adding cv topology folder

### CODE REVIEW

File: MediaGraph/topologies/customvision/readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Analyzing live video using HTTP Extension to send images to an external Azure Custom Vision module 
+
+Azure Custom Vision is an AI service and end-to-end platform for applying computer vision to your specific scenario. It allows you to create a custom computer vision model in minutes. As a customer you can customize and embed state-of-the-art computer vision for specific domains, build frictionless customer experiences, optimize manufacturing processes, accelerate digital marketing campaigns—and more. No machine learning expertise is required.
```
Comment (naiteeks) [inline MediaGraph/topologies/customvision/readme.md:3]: very heavy duty words..."frictionless customer experiences", "optimize manufacturing processes", "accelerate digital marketing campaigns"

File: MediaGraph/topologies/customvision/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+This topology enables you to run Live Video Analytics with Custom Vision on a video feed from an RTSP-capable camera. A subset of the video frames from the camera (as controlled by the Frame Rate Filter) are converted to images, and sent to an external Custom Vision module. The results are then published to the IoT Hub.
```
Comment (naiteeks) [inline MediaGraph/topologies/customvision/readme.md:5]: a RTSP capable camera

File: MediaGraph/topologies/customvision/readme.md (hunk 1 above)
Comment (naiteeks) [inline MediaGraph/topologies/customvision/readme.md:3]: Azure Custom Vision is an AI service and an end-to-end platform

Comment (avkewalr-msft) [reply MediaGraph/topologies/customvision/readme.md]: removed and made it simpler, original was from CV's marketing material

File: MediaGraph/topologies/customvision/readme.md (hunk 2 above)
Comment (avkewalr-msft) [reply MediaGraph/topologies/customvision/readme.md]: changed

File: MediaGraph/topologies/customvision/readme.md (hunk 1 above)
Comment (avkewalr-msft) [reply MediaGraph/topologies/customvision/readme.md]: changed and pushed new commit in

**Comment by naiteeks** [comment]
looks good

**Comment by gadamilan** [comment]
Redundant

**Pull request closed without merging**